tests/infrastructure/prometheus.go

```diff
@@ -640,3 +638,22 @@ func countReadyAndLeaderPods(pod *k8sv1.Pod, component string) (foundMetrics map
 
 	return foundMetrics, err
 }
+
+func GetKubevirtVMMetrics(virtClient *kubecli.KubevirtClient, pod *k8sv1.Pod, ip string) string {
```

Comment:
`kubevirt.Client()` is a singleton, there is no need to pass it as a parameter to the method. The method can simply use `kubevirt.Client()`.